<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveVideoDialog</class>
 <widget class="QWidget" name="SaveVideoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SaveVideoDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="TitleBar">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>39</red>
           <green>51</green>
           <blue>56</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>95</green>
           <blue>135</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>39</red>
           <green>51</green>
           <blue>56</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>95</green>
           <blue>135</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>95</green>
           <blue>135</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>95</green>
           <blue>135</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labelTitlebar">
        <property name="font">
         <font>
          <family>Siemens Sans</family>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
   background-color: rgb(0, 95, 135);
   color: rgb(249, 247, 248 );
   font-size: 20px;
   font-weight: bold;
   font-family: Siemens Sans;
}</string>
        </property>
        <property name="text">
         <string>Save Video</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QFrame" name="frameDashBoardInsertNewVideoFile">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>82</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>82</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
     background:rgb(249,247,248);
     min-height: 80px;
     max-height: 80px;
}

QLabel{
  background:rgb(249,247,248);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 18px;
  max-height: 18px;
}

QLineEdit
{
    background:rgb(255,255,255);
    color:rgb(39,51,66);
    font-size: 16px;
    font-weight: bold;
    font-family: Siemens Sans;
    border: 1px solid rgb(39,51,66); 
    min-height: 32px;
    max-height: 32px;
}
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="rightMargin">
           <number>16</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelMasseageNewVideoFile">
            <property name="font">
             <font>
              <family>Siemens Sans</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Insert New Video File Name:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEditInputDataNewVideoFile">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>34</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>34</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Siemens Sans</family>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="frameDashBoardVideoLocation">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>82</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>82</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
     background:rgb(249,247,248);
     min-height: 80px;
     max-height: 80px;
}

QLabel{
  background:rgb(249,247,248);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 18px;
  max-height: 18px;
}

QLineEdit
{
    background:rgb(255,255,255);
    color:rgb(39,51,66);
    font-size: 16px;
    font-weight: bold;
    font-family: Siemens Sans;
    border: 1px solid rgb(39,51,66); 
    min-height: 32px;
    max-height: 32px;
}
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="rightMargin">
           <number>16</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelMasseageVideoLocation">
            <property name="font">
             <font>
              <family>Siemens Sans</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Video Location:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEditInputDataVideoLocation">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>34</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>34</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Siemens Sans</family>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>16</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>16</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <property name="horizontalSpacing">
        <number>16</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonOK">
         <property name="font">
          <font>
           <family>Siemens Sans</family>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
   background-color: rgb(189,192, 197 );
   color: rgb(39, 51, 66 );
   min-height: 40px;
   font-size: 16px;
   font-weight: bold;
   font-family: Siemens Sans;
  border: none;
}
QPushButton:pressed {
     border: 1px solid rgb(0,95,135);    
 }

</string>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButtonCancel">
         <property name="font">
          <font>
           <family>Siemens Sans</family>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
   background-color: rgb(189,192, 197 );
   color: rgb(39, 51, 66 );
   min-height: 40px;
   font-size: 16px;
   font-weight: bold;
   font-family: Siemens Sans;
  border: none;
}
QPushButton:pressed {
     border: 1px solid rgb(0,95,135);    
 }

</string>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
